lib/types/global.ts

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Does it make sense to add these to ferryman-extensions instead?  That way we can share across components.

Reply:
They actually already are in the extension, I didn't know that!